airbyte-integrations/connectors/source-bigquery/source_bigquery/source.py

```diff
@@ -110,7 +130,10 @@ def check_connection(self, logger, config: Mapping[str, Any]) -> Tuple[bool, any
                         failure_type=FailureType.transient_error,
                         message=error_msg,
                     )
-            return True, error_msg
+            return False, error_msg
+        except Exception as error:
```

Comment:
@maxi297 thanks for the info. Actually I made the modification because in the `check_connection` acceptance test, it expects at least one `Status` message to be emitted. 
I guess it will be better to emit the status message and the raise the error as a system_error in this case